### DISCUSSION TITLE
Energy Dataset Lists

### DISCUSSION THREAD

**Discussion by SteveHaben** [alan-turing-institute/ADViCE#72] [Key sector: Energy]
This list is to share possible datasets for energy problems. This can include lists of infrastructure datasets, telemetry data, smart meter data sets etc. The intention is not to include individual datasets as these can be the topic of individual discussion threads. Instead this thread should make it easier to find multiple datasets of different types which may be required for more extensive modelling.

**Comment by SteveHaben** [comment]
Title: Low Voltage Load Forecasting Data 
Description: This data set consists of low voltage level load data curated by the authors of a review of low voltage load forecasting review. It consists of data from household (smart meters) up to about primary substation level (11kV)
Link: https://low-voltage-loadforecasting.github.io/
Number of datasets at time of writing: 73

**Comment by esc-data-science** [comment]
Here are a few more:

- The UK Energy Data Centre catalogue: https://ukerc.rl.ac.uk/DC/cgi-bin/edc_search.pl 
- Open Energy catalogue: https://openenergy.org.uk/
- Subak Data Catalogue: https://data.subak.org/
- UK Data Service: https://ukdataservice.ac.uk/find-data/
